src/components/Input/styledComponents.tsx

```diff
@@ -0,0 +1,118 @@
+import styled, { css } from "styled-components";
+
+import { SuggestionProps } from "./interface";
+
+import { minDevice } from "../../assets/styles/breakpoints";
+import {
+  white,
+  placeholder,
+  activeButton,
+  lightGrey,
+} from "../../assets/styles/colors";
+import { ReactComponent as LensIcon } from "../../assets/img/icons/search.svg";
+import { ReactComponent as ClearIcon } from "../../assets/img/icons/remove-icon.svg";
+
+const FormWrapper = styled.div`
+  width: 100%;
+  height: 5em;
+  box-shadow: 0px 4px 15px 0px rgba(22, 28, 86, 0.2);
+  z-index: 10;
+  @media ${minDevice.desktopTablet} {
+    width: 22.5em;
+  }
+  @media ${minDevice.desktopL} {
+    width: 31.4375em;
```

Comment:
We can talk about using px, em, rems in particular use cases and come up with some nice `rules` that could help us reflecting our design in a convenient way :)